src/window/window_wrapper/keyboard_manager.rs

```diff
@@ -64,6 +64,7 @@ fn is_control_key(key: Key<'static>) -> Option<&str> {
         Key::End => Some("End"),
         Key::PageUp => Some("PageUp"),
         Key::PageDown => Some("PageDown"),
+        Key::Tab if shift => Some("Tab"),
```

Comment:
Why do we need this `if shift` check?

I tested variant without, and it works fine. Mabe i missed something?